extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/websocket/VertxWebSocketRestHandler.java

```diff
@@ -21,7 +23,7 @@ public void handle(ResteasyReactiveRequestContext requestContext)
     };
 
     @Override
-    public List<ServerRestHandler> handlers(Phase phase) {
+    public List<ServerRestHandler> handlers(Phase phase, ResourceClass resourceClass, ServerResourceMethod resourceMethod) {
```

Comment:
Are you sure about this one? Because the previous implementation covered both methods. The new one gets `handlers(Phase phase)` to return an empty list always.

OK, it's deprecated but who knows who's calling it...

Reply:
Yes, I am sure it's correct. RESTEasy Reactive calls it with the appropriate parameters. 
The old method was introduced when we didn't have a need for those parameters, but now we do (just not in this particular handler) :)

Reply:
OK, you're the boss :).

Reply:
😆